WooCommerce/Classes/ViewRelated/CardPresentPayments/CardReaderConnectionController.swift

```diff
@@ -411,7 +411,7 @@ private extension CardReaderConnectionController {
                 ServiceLocator.analytics.track(.cardReaderSoftwareUpdateCancelTapped, withProperties: analyticsProperties)
                 cancelable.cancel { result in
                     if case .failure(let error) = result {
-                        print("=== error canceling software update: \(error)")
+                        DDLogError("💳 Error: canceling software update \(error)")
```

Comment:
Somehow I learned the British spelling. This always catches me when I see it. No change needed, just funny to me :)

https://oneminuteenglish.org/en/canceling-cancelling/

Reply:
Welcome to my world. Seeing "typos" everywhere 😉